core/jazz_usermanagement/index.js

```diff
@@ -39,17 +39,16 @@ function handler(event, context, cb)  {
 
 	var config = configModule.getConfig(event, context);	
 
-	if (!config || config.length) {
+	if (!config || config.length) { 	
 		logger.error("Cannot load config object, will stop processing");
 		return cb(JSON.stringify(errorHandler.throwInternalServerError("101", "Internal error, please reach out to admins")));
 	}
 
 	global.config = config;
-
 	try {
 		logger.info(JSON.stringify(event));
 
-		if (!event || !event.method || !event.resourcePath) {
+		if (!event || !event.resourcePath) {
```

Comment:
Why do you removing, !event.method. Please don't change the functionality of the service.

Reply:
There are two if, 
Second function only be invoke if its coming out from first. 
As event.method other then POST stuck with first function only and second function will never execute.
So i removed event.method from first function,

Reply:
We need to ensure, event contains "method" key. That is why the first condition for, we have to throw error, event with out method key. 
2nd: While looking the second if you might understand that, wee are handling only POST. Hope it will clear for you.

Reply:
Yes this change updated.